packages/connection-form/src/components/connect-form-actions.tsx

```diff
@@ -41,8 +42,10 @@ function ConnectFormActions({
 }): React.ReactElement {
   return (
     <div className={formActionStyles}>
-      {warnings.length ? <WarningSummary warnings={warnings} /> : ''}
-      {errors.length ? <ErrorSummary errors={errors} /> : ''}
+      {warnings && <WarningSummary warnings={warnings} />}
```

Comment:
I'm not sure checking for `warnings` here makes sense, looking at the code before it seems that `warnings` is always set (`warnings.length` would have thrown otherwise), should we keep `warnings.length` as a guard? otherwise we can remove the guard entirely since you have one inside `WarningSummary`